docs/guide/new-game.md

```diff
@@ -41,7 +41,7 @@ All we have to do is call `rojo build`:
 rojo build -o MyNewProject.rbxlx
 ```
 
-If you open `MyNewProject.rbxlx` in Roblox Studio now, you should see a `Folder` containing a `ModuleScript` under `ReplicatedStorage`!
+If you open `MyNewProject.rbxlx` in Roblox Studio now, you should see a `Folder` containing a `Source` under `ReplicatedStorage`!
```

Comment:
This might be a better wording for this sentence.

```suggestion
If you open `MyNewProject.rbxlx` in Roblox Studio now, you should see a `Folder` named "Source" containing a `ModuleScript` under `ReplicatedStorage`.
```